File: java/src/main/java/com/experian/dvs/client/server/email/RestApiEmailValidateResult.java
package com.experian.dvs.client.server.email;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class RestApiEmailValidateResult {

    @JsonProperty("confidence")
    private String confidence;

    @JsonProperty("did_you_mean")
    private List<String> didYouMean;

    @JsonProperty("verbose_output")
    private String verboseOutput;

    @JsonProperty("email")
    private String email;

    public String getConfidence() {
        return confidence;
    }

    public void setConfidence(String confidence) {
        this.confidence = confidence;
    }

    public List<String> getDidYouMean() {
        return didYouMean;
    }

    public void setDidYouMean(List<String> didYouMean) {
        this.didYouMean = didYouMean;
    }

    public String getVerboseOutput() {
        return verboseOutput;
    }

    public void setVerboseOutput(String verboseOutput) {
        this.verboseOutput = verboseOutput;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: csharp/DVSClient/DVSClient/server/email/RestApiEmailValidateResult.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Email
{
    public class RestApiEmailValidateResult
    {
        [JsonProperty("confidence")]
        public string? Confidence { get; set; }

        [JsonProperty("did_you_mean")]
        public IEnumerable<string>? DidYouMean { get; set; }

        [JsonProperty("verbose_output")]
        public string? VerboseOutput { get; set; }

        [JsonProperty("email")]
        public string? Email { get; set; }
    }
}
